pages/guideforms.py

```diff
@@ -642,6 +642,33 @@ def validate(self, value):
             'https://github.com/WICG/idle-detection/blob/main/HOWTO.md'
             '">Example 2</a>.')),
 
+    'dt_milestone_desktop_start': forms.IntegerField(
+        required=False, label='DevTrail on desktop',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
+                   'this feature on desktop platforms by setting a flag.')),
+
+    'dt_milestone_android_start': forms.IntegerField(
+        required=False, label='DevTrail on Android',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
+                   'this feature on iOS by setting a flag.  '
+                   'Set this only if different from desktop milestone.')),
+
+    'dt_milestone_ios_start': forms.IntegerField(
+        required=False, label='DevTrial on iOS (RARE)',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
+                   'this feature on iOS by setting a flag.  '
+                   'Set this only if different from desktop milestone.')),
```

Comment:
```suggestion
                   'this feature on iOS by setting a flag.  ')),
```

Reply:
This wording implies that iOS is as common to features. I think I can count on one hand the number of times a Chrome Status feature applied to iOS.

Reply:
Removed.